prow/cmd/initupload/BUILD.bazel

```diff
@@ -1,11 +1,10 @@
-load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+load("@io_bazel_rules_docker//container:image.bzl", "container_image")
 load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
 
-go_image(
+container_image(
     name = "image",
     base = "@alpine-base//image",
-    binary = ":initupload",
-    directory = "/",
```

Comment:
why did you remove `directory` here but not in the other images?

Reply:
I'm having some trouble with buildifier-on-save eating my changes :confused:, I put this back... fixing

Reply:
fixed, we definitely want that.